tests/devices/test_local_decoder_padding.py

```diff
@@ -0,0 +1,89 @@
+from roborock.protocol import create_local_decoder, create_local_encoder
```

Comment:
We currently have the pattern that each test module corresponds to an actual model. I know AI likes to make up random test files depending on the subject, but i'd rather see this correspond to a module.

Should this be in `tests/test_protocol.py`?